CMakeLists.txt

```diff
@@ -4,27 +4,60 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+include(GNUInstallDirs)
 include(ThirdParties)
 include(ClangTidy)
 include(PedanticCompiler)
 
 set(boxed_cpp_HEADERS
-    include/boxed-cpp/boxed.hpp
+    ${PROJECT_SOURCE_DIR}/include/boxed-cpp/boxed.hpp
```

Comment:
Should we adapt this for our other libraries as well? Especially libunicode? What do you think?

Reply:
I think it is fine if you want to use `${PROJECT_SOURCE_DIR}` in this situation for libunicode.